### DISCUSSION TITLE
Unable to deploy my django web application on azure

### DISCUSSION THREAD

**Discussion by akj1608** [github/india#803] [Students]
I have followed the azure app service documentation but wasn't able to deploy my app and build failed drastically.. please if anyone can help me out with this😔.. Just due to this reason my project was rejected and don't want to ruin my past one month work.. please assist me if you can

**Comment by ey-mentor1** [comment]
Hello,

Can you please share the error and basic details, so that we can look into the issue faced by you.

Thank You

**Comment by akj1608** [comment]
https://github.com/akj1608/bicaption/runs/5498451909?check_suite_focus=true

https://github.com/akj1608/bicaption


https://frts.azurewebsites.net/

**Comment by ey-mentor1** [comment]
Hello,
Issue is that the python is in /opt
And as per the error the files is nit found in /tmp folder path.
Can you please recheck the deployment yaml file.
And follow the paths and steps mentioned in this link
https://docs.microsoft.com/en-us/azure/app-service/deploy-github-actions?tabs=applevel
Thank you
